frontend/src/libs/components/tags/tags.tsx

```diff
@@ -12,7 +12,13 @@ type Properties = {
 };
 
 const Tags: React.FC<Properties> = ({ className, tags }) => (
-  <ul className={getValidClassNames(styles.tags, className)}>
+  <ul
+    className={getValidClassNames(
+      styles.tags,
+      className,
+      tags.length === 1 && styles.singleTag,
```

Comment:
B31

Reply:
can I name constant SINGLE_NUMBER? Have no ideas how to name better